starter/scripts/modules/enable_camera.dart

```diff
@@ -66,10 +66,30 @@ ensemble_camera:
       updateIOSPermissions(iOSPermissions, arguments);
     }
 
+    // Add the face detection models to the web/index.html file
+    if (platforms.contains('web')) {
+      const webIndexHtml = '''
+<!-- Face Detection Scripts -->
```

Comment:
use a different comment than the comment to look for for doing the replacement. this way if you run the script multiple times, you won't get multiple entries.